src/common/enums/network-names.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum NetworkNames {
+  MAINNET = 'homestead',
```

Comment:
I'm just curious, is 'homestead' a synonym of 'mainnet'?

Reply:
I also ready this in stackoverflow :P I wanted just to make sure the value 'homestead' wasn't going to change in a future to be called 'mainnet'